trio-stubs/lowlevel.pyi

```diff
@@ -158,15 +176,21 @@ class ParkingLot:
 class _RunVarToken:
     pass
 
-class RunVar(Generic[T]):
-    def __init__(self, name: str, default: T = ...) -> None: ...
-    def get(self, default: T = ...) -> T: ...
-    def set(self, value: T) -> _RunVarToken: ...
+@final
+@attr.s(eq=False, hash=False, slots=True)
+class RunVar(Generic[_T], metaclass=ABCMeta):
+    _NO_DEFAULT = object()
```

Comment:
I don't think stubtest should be insisting that we expose underscore-prefixed attributes...

Reply:
The idea is that these attributes don't exist in the real thing, but if we say they do in the stubs, typecheckers think that they do exist, which confuses everyone. It is best practice to underscore thinks that don't exist in the real version.

Reply:
It's not really stubtest's fault here. It needs these attributes to construct the correct `__init__` with attrs. The error only complains about `__init__` and `__match_args__`, not missing attributes. I can remove the `_NO_DEFAULT` and just inline the `object()` however.